docs/source/user/fast.farm/ModelGuidance.rst

```diff
@@ -513,6 +513,12 @@ frequency of wake meandering should be resolved by at least :math:`10`
 time steps. Note that :math:`D^\text{Wake}` can be approximated as
 :math:`D^\text{Rotor}` in this calculation.
 
+When **Wake_Mod=2,3**, for numerical stability, it is recommended to set the time step with a value that (approximately) satisfies the following guideline (see Equation 20 of the following `paper <https://doi.org/10.5194/wes-6-555-2021>`__):
+
+   .. math::
+      \textbf{DT_Low}  \gtrapprox \frac{2 \textbf{dr }}{V_\text{Hub}}
```

Comment:
I'm not sure I understand how this relates to Equation (20) in the referenced paper, but the text after Equation (20) seems to be used.  I derive:

dy = D/10 --> dr = D/10

dx = D/20 = V_hub*DT_Low --> DT_Low = dr / ( 2*Vhub )

But this differs from what is stated.  And why is the RHS a lower bound?  Isn't a smaller time step more stable than a large time step?

Reply:
Good catch @jjonkman , yes, your derivations are correct.

This can be changed to:
`\textbf{DT_Low}  \lessapprox \frac{ \textbf{dr }}{2 V_\text{Hub}}`